comploinc_config.yaml

```diff
@@ -27,11 +27,15 @@ loinc_nlp_tree:
 
 snomed:
   release:
-    default: 'INT_20240801'
+    default: 'INT_20250101'
```

Comment:
Reminder: @ShahimEssaid or @joeflack4 to upload an updated sources `.zip` to the [sources folder](https://drive.google.com/drive/folders/1wguwYdsasYmKL_hSOqW0yeN7ICw4LgLV).